marti_common_msgs/msg/HealthStatus.msg

```diff
@@ -0,0 +1,4 @@
+Header header
+
+bool   healthy
+string message
```

Comment:
Well, it's our message, so we can make our own concrete definitions.

- `OK`: Operating normally, no problems
- `WARN`: Component is operating but may be producing incorrect output or having issues
- `ERROR`: Component has encountered a fatal error and cannot or will not operate properly
- `UNKNOWN`: The module producing the report is unable to determine the status (due to timeouts or not being initialized yet), or a module that is relaying status information for another component has not yet received any status messages